build_library/write_hdd.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Generate a parallels compatible disk with XML format. qemu-img does not currently
+# support the XML disk format. If changes in the future this script can replaced simply
+# by a qemu-img convert.
+
+SCRIPT_ROOT=$(readlink -f $(dirname "$0")/..)
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+
+DEFINE_string input_disk_image "" "Disk image to convert from, required."
+DEFINE_string input_disk_format "raw" "Disk image format."
+DEFINE_string output_disk "" "Path to the output disk, required."
+
+# Parse command line
+FLAGS "$@" || exit 1
+eval set -- "${FLAGS_ARGV}"
+
+# Die on any errors.
+switch_to_strict_mode
+
+if [[ ! -e "${FLAGS_input_disk_image}" ]]; then
+    echo "No such disk image '${FLAGS_input_disk_image}'" >&2
+    exit 1
+fi
+
+rm -fr "${FLAGS_output_disk}"
+mkdir -p "${FLAGS_output_disk}"
+disk_name=$(basename "${FLAGS_output_disk}")
+
+touch "${FLAGS_output_disk}"/"${disk_name}"
+
+snapshot_uuid="{5fbaabe3-6958-40ff-92a7-860e329aab41}"
```

Comment:
Is this supposed to be static? If so add a comment what it actually means, if not call uuidgen